website/content/en/docs/building-operators/golang/project_migration_guide.md

```diff
@@ -1,182 +1,315 @@
-
-The project can now be built, and the operator can be deployed on-cluster. For further steps regarding the deployment of the operator, creation of custom resources and cleaning up of resources, refer to the [quickstart guide][kb_quickstart].
-
-
-[memcached-operator]:/docs/building-operators/golang/quickstart/
-[git_tool]: https://git-scm.com/downloads
-[go_tool]: https://golang.org/dl/
-[docker_tool]:https://docs.docker.com/install/
-[kubectl_tool]: https://github.com/kubernetes/minikube#installation
-[kustomize_tool]: https://github.com/kubernetes-sigs/kustomize/blob/master/docs/INSTALL.md
-[kubebuilder_install]: https://book.kubebuilder.io/quick-start.html#installation
-[memcached_controller]: https://github.com/operator-framework/operator-sdk/blob/8323f56b91590c3bc8098e0024aa825e95386c8a/example/memcached-operator/memcached_controller.go.tmpl
-[sdk_reconcile_struct]: https://github.com/operator-framework/operator-sdk/blob/8323f56b91590c3bc8098e0024aa825e95386c8a/example/memcached-operator/memcached_controller.go.tmpl#L74-L80
-[generate_crd]: https://book.kubebuilder.io/reference/generating-crd.html?highlight=make,mani#generating-crds
-[deployment_yaml]: https://github.com/operator-framework/operator-sdk-samples/blob/master/go/memcached-operator/deploy/operator.yaml
+   	. "github.com/onsi/ginkgo"
+   	. "github.com/onsi/gomega"
+...
+)  
+```                     
+
+SDK has been adopted this stack because it is used in the Kubernetes universe for the projects used by it. To know more about how you can test your controllers see [Writing controller tests][writing-controller-tests]. However, you still able to use the libs that please you to write tests.  
+
+## Migrating your CR's
+
+Now, the CR's are generated in the `/config/samples` so we will need to update them with
+the specs required. In our example, we'll copy the specs from `deploy/crds/cache.example.com_v1alpha1_memcached_cr.yaml` 
+to `config/samples/cache_v1alpha1_memcached.yaml`
+
+### Configuring your Operator
+
+In case your project has customizations in the `deploy/operator.yaml` then, it needs to be port to 
+`config/manager/manager.yaml`. However, note that the following env vars are no longer used. 
+
+- `OPERATOR_NAME` is deprecated. It is used to define the name for a leader election config map. Operator authors should begin using `--leader-election-id` instead.
+- `POD_NAME` was used to enable a particular pod to hold the leader election lock when the Helm operator used the leader for life mechanism. Helm operator now uses controller-runtime's leader with lease mechanism, and `POD_NAME` is no longer necessary.
+
+## Exporting metrics 
+
+If you are using metrics and would like to keep them exported see that the `addMetrics()` function is no longer 
+generated in the `main.go`. To export the metrics you will need to configure it in the 
+`config/default/kustomization.yaml`. Please see the [metrics][metrics] doc to know how you can perform this setup. 
```

Comment:
Most of the metrics setup is already done in `config/default/kustomization.yaml`. The only additional piece that users might want is the setup of the Prometheus `ServiceMonitor`. Let's be explicit about these details.